.storybook/polaris-readme-loader.js

```diff
@@ -44,13 +48,15 @@ export function ${example.storyName}() {
       example.storyName
     }Component /></div>;
 }
-${example.storyName}.story = {
-  name: ${JSON.stringify(example.name)},
-  decorators: [withA11y],
-  parameters: {
-    notes: ${JSON.stringify(example.description)},
-    percy: {skip: ${JSON.stringify(!testIndividualExamples)}},
-  }
+
+${example.storyName}.storyName = ${JSON.stringify(example.name)};
+${example.storyName}.args = {omitAppProvider: ${readme.omitAppProvider}};
+${example.storyName}.parameters = {
+  layout: '${hasFullscreenLayout ? 'fullscreen' : 'padded'}',
```

Comment:
The new layout parameter controls if a story renders flush to the edge ("full screen") or has a 1rem of padding ("padded"), which removes the need for our PaddingDecorator helper